contrib/pa-rekey

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# rotate keys and reencrypt passwords
+
+basedir="${XDG_DATA_HOME:=$HOME/.local/share}/pa"
+: "${PA_DIR:=$basedir/passwords}"
+
+# Restrict permissions of any new files to only the current user.
+umask 077
+
```

Comment:
```suggestion
cp "$basedir/identities" "$basedir/identities.tmp"
cp "$basedir/recipients" "$basedir/recipients.tmp"
```

Reply:
maybe parametrize if we want to replace all existing identities and recipients or adding new identity

Reply:
I think replacing keys makes more sense as a default, with adding keys as a possible option. Imagine you want to rekey because your identities file has been compromised, you'd want pa-rekey not to reuse it

Reply:
i don't care about the defaults. just want to have somehow the possibility to add a new recipient and re-encrypt all my passwords with all the recipients

Reply:
it's all good, then, it's possible